keyboards/kradoindustries/krado66/keymaps/vial/rules.mk

```
@@ -0,0 +1,12 @@
+VIA_ENABLE = yes
+LTO_ENABLE = yes
+VIAL_ENABLE = yes
+VIAL_ENCODERS_ENABLE = yes
+COMBO_ENABLE = no
+QMK_SETTINGS = yes
+ENCODER_ENABLE = yes
+ENCODER_MAP_ENABLE = yes
+RGB_MATRIX_DRIVER = WS2812
+KEY_OVERRIDE_ENABLE = no
+TAP_DANCE_ENABLE = no
+MOUSEKEY_ENABLE = no
```

Obsolete and redundant rules removed.
```suggestion
COMBO_ENABLE = no
QMK_SETTINGS = yes
ENCODER_MAP_ENABLE = yes
KEY_OVERRIDE_ENABLE = no
TAP_DANCE_ENABLE = no
MOUSEKEY_ENABLE = no
```